### PR TITLE
Created Checkpoint Golf Code

### PR DESCRIPTION
**Pull request by osterbergmarcus**
Created Checkpoint-Golf-Code help article

### CODE REVIEW

**Comment by hallaathrad** [comment]
Would you please change the file title and equivalents from Checkpoint to Waypoint?

File: Checkpoint-Golf-Code.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Details
+We will now use our knowledge about else if statements and comparison with equality, less and greater operators.
```
Comment (SaintPeter) [inline Checkpoint-Golf-Code.md]: Missing the full description and table.

File: Checkpoint-Golf-Code.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    return "Eagle";
+  } else if (strokes == par -1) {
+    return "Birdie";
+  } else if (strokes == par) {
+    return "Par";
+  } else if (strokes == par +1) {
+    return "Bogey";
+  } else if (strokes == par +2) {
+    return "Double Bogey";
+  } else if (strokes >= par +3) {
```
Comment (SaintPeter) [inline Checkpoint-Golf-Code.md]: Do not need the last if statement, just an else.

File: Checkpoint-Golf-Code.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    return "Go Home!";
+  } else {
+  return "Change Me";
```
Comment (SaintPeter) [inline Checkpoint-Golf-Code.md]: This line should be removed.

File: Checkpoint-Golf-Code.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  // Only change code above this line
+}
+}
+// Change these values to test
+golfScore(5, 4);
+```
+
+# Code Explanation:
+- Comparing the `parameters` stroke and par value to return appropriated string value.
+- Using else if statement for flow control
```
Comment (SaintPeter) [inline Checkpoint-Golf-Code.md]: Need better explanation of why you can just have an else at the end.

File: Checkpoint-Golf-Code.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+## Useful Links
+- [Waypoint: Chaining If Else Statements](http://www.freecodecamp.com/challenges/waypoint-chaining-if-else-statements)
+- [Waypoint: Comparison with the Greater Than Equal To Operato](http://www.freecodecamp.com/challenges/waypoint-comparison-with-the-greater-than-equal-to-operator)
+- [Waypoint: Comparison with the Less Than Equal To Operator](http://www.freecodecamp.com/challenges/waypoint-comparison-with-the-less-than-equal-to-operator)
+
+## Problem Explanation:
+- Change the code below `// Only change code below this line` and above `// Only change code above this line`.
+- Take note that you are editing the inside of the `golfScore` function.
+- You will have to make the function return a specific string, depending on the value of the parameters `par` and `strokes` that are passed in to your function.
+
```
Comment (SaintPeter) [inline Checkpoint-Golf-Code.md]: Maybe a note about needing EXACT strings.  This is a common error.

**Comment by SaintPeter** [comment]
Please see inline comments.  This article needs work before it can be accepted.

**Comment by osterbergmarcus** [comment]
@SaintPeter Thanks! I've improved the article :+1:

**Comment by SaintPeter** [comment]
Lookin' good.  Thanks!

**Pull request merged**